gateway-home/modules/ROOT/pages/index.adoc

```diff
@@ -1,53 +1,85 @@
-
-Anypoint Platform offers three different runtime options for managing and securing your APIs.
+* <<anypoint_flex_gateway,Anypoint Flex Gateway>>
+* <<anypoint_mule_gateway,Anypoint Mule Gateway>>
 
+[[anypoint_flex_gateway]]
 == Anypoint Flex Gateway
 
-Anypoint Flex Gateway is an ultrafast lightweight API gateway designed to manage and secure APIs running anywhere.
+Anypoint Flex Gateway is an Envoy-based, ultrafast lightweight API gateway designed to manage and secure APIs running anywhere.
 Built to seamlessly integrate with DevOps and CI/CD workflows, Anypoint Flex Gateway delivers the
 performance required for the most demanding applications and microservices while providing enterprise
 security and manageability across any environment.
 
-For more information, see xref:gateway::flex-gateway-getting-started.adoc[Getting Started with Flex Gateway].
+For more information, see xref:gateway::index.adoc[].
+
+//For more information, see xref:gateway::flex-gateway-getting-started.adoc[Getting Started with Flex Gateway].
 
+[[anypoint_mule_gateway]]
 == Anypoint Mule Gateway
 
-Mule Runtime includes an embedded Mule Gateway. Using this gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
+Mule runtime engine includes an embedded Mule Gateway. Mule Gateway enables you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
 
-Overall, Mule Gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
+Using Mule Gateway, any user can, for example, apply a basic authentication policy on top of a Mule application, enrich an incoming/outgoing message, or add any other complex capability to an API without having to write any code.
 
 If you want to apply policies and generate analytics info, you must use either:
 
 * An API defined in a Mule application.
 * A Mule application that proxies your existing backend implementation.
 
-For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[Mule Gateway Capabilities].
+For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[].
+
+== Flex Gateway vs. Mule Gateway
+
+While Flex Gateway is a high-performance Envoy-based gateway with a small footprint, MuleSoft also provides a simple Mule Gateway for Mule applications.
```

Comment:
Now this feels a bit redundant, coming right after the full-on descriptions. You don't need to repeat the previous info here. Suggest updating the descriptions of each with any info from here that's not already captured. And then focus this section on the comparison between the two.

Reply:
Good point! I think I fixed it.